build/README.md

```diff
@@ -0,0 +1,24 @@
+# Convox Build Action
+This Action [builds](https://docs.convox.com/deployment/builds) an app based on a [convox.yml](https://docs.convox.com/application/convox-yml) so that app can be deployed on Convox
+
+## Inputs
+### `password`
+**Required** The value of your [Convox Deploy Key](https://docs.convox.com/console/deploy-keys)
+### `host`
```

Comment:
The optional parameter in the middle of required parameters will probably go unnoticed. Should generally list all required parameters first unless there is some other (obvious?) ordering, e.g. alphabetical. This will be better when moved out to a login step.